src/intrinsics/SetPrototype.mts

```diff
@@ -172,6 +232,7 @@ export function bootstrapSetPrototype(realmRec) {
     ['has', SetProto_has, 1],
     ['size', [SetProto_sizeGetter]],
     ['values', SetProto_values, 0],
+    ['union', SetProto_union, 1],
```

Comment:
I'm actually fairly unsure what the number here means, so please double check this. :)

Reply:
its the arity of the function. based on this in the spec:

> Every built-in [function object](https://tc39.es/ecma262/#function-object), including [constructors](https://tc39.es/ecma262/#constructor), has a "length" property whose value is a non-negative [integral Number](https://tc39.es/ecma262/#integral-number). Unless otherwise specified, this value is the number of required parameters shown in the subclause heading for the function description. Optional parameters and rest parameters are not included in the parameter count.

> [NOTE 2] For example, the [function object](https://tc39.es/ecma262/#function-object) that is the initial value of the "map" property of the [Array prototype object](https://tc39.es/ecma262/#sec-properties-of-the-array-prototype-object) is described under the subclause heading «Array.prototype.map (callbackFn [ , thisArg])» which shows the two named arguments callbackFn and thisArg, the latter being optional; therefore the value of the "length" property of that [function object](https://tc39.es/ecma262/#function-object) is 1𝔽.